src/coreclr/vm/assembly.cpp

```diff
@@ -1640,119 +1640,6 @@ BOOL Assembly::GetResource(LPCSTR szName, DWORD *cbResource,
     return result;
 }
 
-#ifdef FEATURE_READYTORUN
-BOOL Assembly::IsInstrumented()
-{
-    STATIC_CONTRACT_THROWS;
-    STATIC_CONTRACT_GC_TRIGGERS;
-    STATIC_CONTRACT_FAULT;
-
-    // This will set the value of m_isInstrumentedStatus by calling IsInstrumentedHelper()
```

Comment:
Can we also remove `m_isInstrumentedStatus` and `IsInstrumentedStatus` now?

Reply:
Yes. I intended to to that, and got... distracted deleting other stuff...